packages/nc-gui/components/account/Token.vue

```diff
@@ -89,6 +90,12 @@ const deleteToken = async (token: string): Promise<void> => {
 }
 
 const generateToken = async () => {
+  if (!selectedTokenData.value.description?.length) {
+    isTextEmpty.value = true
+  } else {
+    isTextEmpty.value = false
+  }
+  if (isTextEmpty.value) return
```

Comment:
any warning message needs to be popped up? Like telling users the description is missing.